src/key_packages/mod.rs

```diff
@@ -237,6 +237,15 @@ pub struct KeyPackageBundle {
     pub(crate) private_key: HPKEPrivateKey,
 }
 
+impl Clone for KeyPackageBundle {
+    fn clone(&self) -> Self {
+        KeyPackageBundle {
+            key_package: self.key_package.clone(),
+            private_key: HPKEPrivateKey::new(self.private_key.as_slice().to_vec()),
```

Comment:
Can you add a comment here that this requires the hpke hazmat feature to make clear that this might be undesirable?